### PR TITLE
Face tracking

### PR DESCRIPTION
**Pull request by gferraro**

### CODE REVIEW

File: webserver/static/js/face.ts [hunk 1]
```diff
@@ -0,0 +1,698 @@
...
+      return this.roi.width();
+    }
+    return 0;
+  }
+
+  active(): boolean {
+    return this.framesMissing < maxFrameSkip;
+  }
+
+  update(forehead: ROIFeature | null, roi: ROIFeature | null) {
```
Comment (hardiesoft) [inline webserver/static/js/face.ts:377]: Typing of this seems problematic, if these args can indeed be null, then how can you assume they're not null in the body of the function?

File: webserver/static/js/feverscreen.ts [hunk 2]
```diff
@@ -34,7 +39,7 @@ let RefRadius = 0;
 
 // Load debug mode, if set
 let dbg = window.localStorage.getItem("DEBUG_MODE");
-let DEBUG_MODE = false;
+let DEBUG_MODE = true;
```
Comment (hardiesoft) [inline webserver/static/js/feverscreen.ts]: I don't think we want debug mode to be on by default?

File: webserver/static/js/face.ts [hunk 3]
```diff
@@ -0,0 +1,698 @@
+import {
```
Comment (hardiesoft) [inline webserver/static/js/face.ts:1]: Not going to hold back from getting this merged, but I'd like to have a high-level overview of how the algorithm works at some stage added as a comment.

**Review by hardiesoft — CHANGES_REQUESTED** [review]
Just a couple of wee things to amend, then I think we're good to go and get people testing this, and making videos to improve it.

File: webserver/static/js/face.ts (hunk 1 above)
Comment (gferraro) [reply webserver/static/js/face.ts:377]: thats what this.tracked() does

**Review by hardiesoft — APPROVED** [review]
Looks good to me!

**Pull request merged**